keps/1136-provisioning-request-support/README.md

```diff
@@ -77,6 +78,21 @@ There doesn't seem to be much risks or mitigations.
 [Two phase admission process](https://github.com/kubernetes-sigs/kueue/tree/main/keps/993-two-phase-admission)
```

Comment:
this paragraph is no longer true, you can remove it.

Reply:
Done